### ISSUE TITLE
No package available for ubuntu

### ISSUE THREAD

**Issue by hboetes** [theforeman/foreman_monitoring#36]
The docs suggest I can install this package on xenial ubuntu: ruby-foreman_monitoring, but this is not the case:

```
# sudo apt-get update|grep plugin        
Ign:13 http://deb.theforeman.org plugins InRelease
Hit:17 http://deb.theforeman.org plugins Release
# apt-get install ruby-foreman_monitoring
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package ruby-foreman_monitoring
```

**Comment by timogoebel** [comment]
@mmoll: Can you jump in here and help? Thanks.

**Comment by mmoll** [comment]
Debian does not allow underscores, so dashes are used there instead:

```
mmoll@sledge:~$ apt-cache madison ruby-foreman-monitoring
ruby-foreman-monitoring |    0.1.1-1 | http://deb.theforeman.org plugins/nightly amd64 Packages
```

**Comment by hboetes** [comment]
@timogoebel can you please update the docs to reflect the proper name?

**Comment by timogoebel** [comment]
@hboetes: Fixed in e8c7a5ca7803024e9c80a979da0213c60cb644e0, thanks.
